.github/workflows/rust.yml

```diff
@@ -125,6 +125,26 @@ jobs:
           command: audit
           args: --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2022-0040
 
+  spellcheck:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v2
+      - name: Install Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          override: true
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v1.3.0
+      - name: Install spellcheck
+        run: cargo install cargo-spellcheck
+      - name: Run Spellcheck
+        run: |
```

Comment:
Seems a bit manual, why not just `cargo spellcheck --code 1 --recursive --cfg .config/spellcheck.toml blockchain encoding forest`?

Reply:
I was unsure if that would work. `cargo spellcheck --recursive list-files blockchain` shows nothing, indicating that it won't traverse the sub-directories. Running `cargo spellcheck --recursive list-files blockchain/beacon` shows the expected output.

However, I tested it and it appear to work even though not all the files are shown by `list-files`.